docs/immortal-empires/high-elves.md

```diff
@@ -0,0 +1,41 @@
+# High Elves
+
+## Tyrion - Eataine
+
+* **Route I - Defender of Ulthuan**
```

Comment:
I would include the requirement of controlling the whole donut, inclduing military allies. It should be something easy to achieve and feels very loreful, especially with the economic reward.

For the reward, I would go 100% economic -> 
- _The Flourishing of the Ten Kingdoms_: Control: +4, Growth: 60, Income from Ports: +50%

Reply:
My only concern is this will be Alarielle's objective since her faction mechanic is controlling the donut. So Alarielle and Tyrion will be very similar.

Reply:
Fair point, but considering we got six High Elves factions, is it normal to expect some overlap. Even more if these two start so close to each other in a deffensive position like the donut. 

In general, for order races, I like the racial leader to have an objective about unifying its lands: We do the same with The Empire, Kislev, Cathay, Dwarfs and Bretonnia. Even Dark Elves and Lizardmen follow similar objectives.

I would make Allariele first route about Hellborn and second route about the donut, just for the sake of minor difference. But it is not a big deal IMO.